sentry/vendor_images.yaml

```diff
@@ -1,7 +1,7 @@
 docker.io:
   images-by-tag-regex:
-    getsentry/relay: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
-    getsentry/sentry: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
-    getsentry/snuba: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
-    getsentry/symbolicator: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
-    yandex/clickhouse-server: ^v?[0-9]{2}\.[0-9]+?\.[0-9]+$
+    getsentry/relay: (?:^2[3-9]\.[0-9]+?\.[0-9]+$)|(?:^[3-9][0-9]\.[0-9]+?\.[0-9]+$)
+    getsentry/sentry: (?:^2[3-9]\.[0-9]+?\.[0-9]+$)|(?:^[3-9][0-9]\.[0-9]+?\.[0-9]+$)
+    getsentry/snuba: (?:^2[3-9]\.[0-9]+?\.[0-9]+$)|(?:^[3-9][0-9]\.[0-9]+?\.[0-9]+$)
+    getsentry/symbolicator: (?:^2[3-9]\.[0-9]+?\.[0-9]+$)|(?:^[3-9][0-9]\.[0-9]+?\.[0-9]+$)
+    yandex/clickhouse-server: (?:^2[2-9]\.[0-9]+?\.[0-9]+$)
```

Comment:
You can remove clickhouse from the image vendoring since those are handled by the operator application now.